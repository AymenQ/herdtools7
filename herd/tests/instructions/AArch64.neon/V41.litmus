AArch64 V41
(* Tests LDP - load pair without offset *)
{ 
0:X0=x; 0:X1=y; 0:X2=z;
}

 P0                     ;
 MOV X3, #1             ;
 MOV X4, #2             ;
 STR X3, [X0]           ;
 STR X4, [X0, #4]      ;
 LDP S0, S1, [X0]       ;
 STR S0, [X1]           ;
 STR S1, [X2]           ;
 forall (y=1 /\ z=2)
