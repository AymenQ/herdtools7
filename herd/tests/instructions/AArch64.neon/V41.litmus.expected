Test V41 Required
States 1
y=1; z=2;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (y=1 /\ z=2)
Observation V41 Always 1 0
Hash=90438ca99f247efd837b055fef4bfdde

